api/index.js

```diff
@@ -61,6 +62,7 @@ module.exports = async (req, res) => {
       hide_title: parseBoolean(hide_title),
       hide_border: parseBoolean(hide_border),
       hide_rank: parseBoolean(hide_rank),
+      show_forks: parseBoolean(show_forks),
```

Comment:
This is not scalable. we have to do something like: show_stats=forks,extra,extra

Reply:
Okay

Reply:
done